.travis.yml

```diff
@@ -26,3 +26,6 @@ cache:
   yarn: true
   directories:
   - node_modules
+
+notifications:
+  email: false
```

Comment:
I think you were just joking, but I turned it off because the tests have to pass before merging anyway